servers.json

```diff
@@ -158,6 +158,14 @@
             "hycraft.us"
         ]
     },
+    {
+        "id": "minelabs",
+        "name": "MineLabs",
+        "addresses": [
+            "play.minelabs.org"
```

Comment:
You do not need to add sub-domains to the addresses array.

Refer to the [ip-addresses](https://github.com/LunarClient/ServerMappings#ip-addresses) section of the README for more information.